business/checkers/no_service_checker.go

```diff
@@ -23,6 +24,8 @@ func (in NoServiceChecker) Check() models.IstioValidations {
 		return validations
 	}
 
+	log.Infof("ServiceEntries: %v\n", in.IstioDetails.ServiceEntries)
```

Comment:
Debug trace instead ?
It sounds odd as we don't have these info logging on these area, I wonder if it was intentional.